src/global/global_context.cc

```diff
@@ -34,3 +34,4 @@ const char *g_assert_condition = 0;
 unsigned long long g_assert_thread = 0;
 char g_assert_thread_name[4096];
 char g_assert_msg[8096];
+char g_process_name[4096];
```

Comment:
might want to initialize it to "unknown" or an empty string.

Reply:
Static vars are always initialized with zeroes, so it is already an empty string.